docker-compose.override.yml

```diff
@@ -0,0 +1,5 @@
+version: '3.4'
```

Comment:
I'm guessing you didn't intend to include this file in your commit? :)

Reply:
Initially, I'd meant to; IIRC, when I'd created this branch initially, the port was exposed on the `web` service, which implied that the port exposure was a setting not meant for production (e.g., in production, that port would be closed off in favor of running behind a proxy, and the port mapping would be in developer overrides in `docker-compose.override.yml`). 

The main reason I'd added this in the first place is that an override file can't remove an existing port mapping, and wanted the public port exposure to be optional for the fairly common use-case of running behind a proxy without exposing a redundant port.

What are your thoughts on that? If you'd like to keep that mapping in the main `docker-compose.yml` file, would you be willing to make the publicly-exposed port configurable?